### ISSUE TITLE
Cant install 1.2.1 after accidntally installing 1.4 on fresh system.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mattscotthudson/soundflower#5]
```
What steps will reproduce the problem?
1. I accidentally installed 1.4 without previously having installed any version.
2. Searched for Soundflower.kext - no such thing on my computer.
3. Tried to uninstall using 1.2.1 uninstaller script - get error 
"Soundflower.kext doesnt exist". 
4. Tried to install 1.2.1 but getredmarked harddrives and error "Cant installed 
because newer 
version is already installed"

What is the expected output? What do you see instead?
I expect to be able to either uninstall 1.4 or downgrade to 1.2.1. 

What version of the product are you using? On what operating system?
1.4 and 1.2.1. MacBook Pro 2.6Ghz Intel Core 2 Duo, Leopard 10.5.2.

Please provide any additional information below.

There is a similar issue a few days ago, electrotap posted a fix but I still 
have this problem 
(maybe coming installer will have the new fix implemented?).





```

Original issue reported on code.google.com by `mateuszh...@gmail.com` on 1 Nov 2008 at 10:28

**Comment by GoogleCodeExporter** [comment]
```
the same problem!
there was problems with ver. 1.4, so i decided to make downgrade to 1.2.1 - so 
i removed using uninstall 
command from soundflower 1.2.1 bundle, but now i can't install it again, cause: 
"...newer version is already 
installed". how to uninstall this 1.4 vers? 
```

Original comment by `idmm...@gmail.com` on 2 Nov 2008 at 2:38

**Comment by GoogleCodeExporter** [comment]
```
peut-être faut-il supprimer aussi 
 /library/receipts/soundflower.pkg
avant de reinstaller la version 1.2

delete also before re-install 1.2
 /library/receipts/soundflower.pkg
```

Original comment by `jean.fra...@orange.fr` on 2 Nov 2008 at 11:11

**Comment by GoogleCodeExporter** [comment]
```
thank u a lot!
```

Original comment by `idmm...@gmail.com` on 3 Nov 2008 at 8:58

**Comment by GoogleCodeExporter** [comment]
```
This is resolved with the new Soundflower 1.4.1 release, which is now available.
```

Original comment by `74obje...@gmail.com` on 14 Nov 2008 at 6:04
- Changed state: **Fixed**
